samples/sensor/hx711/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
```

Comment:
Please move the sample to a separate commit. You can add the commit to this PR, don't need to create a new PR.

Reply:
Done.

Reply:
Thanks